example/styles/globals.css

```diff
@@ -17,9 +17,11 @@ body::before {
   bottom: 0;
   left: 0;
   background: url('/background.svg') no-repeat;
+  background-position: 0px 250px;
   width: 100%;
   height: 100%;
   background-size: cover;
+  opacity: 0.6;
```

Comment:
I have reduced this opacity and moved the image down for better text readability.